onadata/apps/api/viewsets/data_viewset.py

```diff
@@ -347,6 +347,12 @@ def list(self, request, *args, **kwargs):
             tags = self.request.query_params.get('tags')
             not_tagged = self.request.query_params.get('not_tagged')
 
+            self.object_list = filters.InstanceFilter(
```

Comment:
I would have expected this to be added to the `DataViewSet.filter_backends` property. Why is this not the case?